.github/workflows/ci-tests-app.yml

```diff
@@ -124,8 +124,7 @@ jobs:
         AWS_DEFAULT_REGION: us-east-1
       run: |
         python -m coverage run --source ${{ env.COVERAGE_SCOPE }} \
-          -m pytest -m "not cloud" tests_app \
-          --timeout=300 -vvvv --durations=50
+          -m pytest -m "not cloud" tests_app --timeout=300 -vvvv --durations=50 --random-order-seed=42
```

Comment:
You could remove the randomization from this workflow and get it merged for fabric and pl to unblock

Reply:
```suggestion
          -m pytest -m "not cloud" tests_app --timeout=300 -vvvv --durations=50
```